src/components/Tool/Appeal/AppealProgressBar.test.tsx

```diff
@@ -0,0 +1,132 @@
+import { ThemeProvider } from '@mui/material/styles';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
+import { render, waitFor } from '@testing-library/react';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { AppealsWrapper } from 'pages/accountLists/[accountListId]/tools/appeals/AppealsWrapper';
+import theme from 'src/theme';
+import AppealProgressBar, { AppealProgressBarProps } from './AppealProgressBar';
+
+const router = {
+  query: { accountListId: 'aaa' },
+  isReady: true,
+};
+
+const Components = ({
+  given,
+  received,
+  committed,
+  amount,
+  amountCurrency,
+}: AppealProgressBarProps) => (
+  <LocalizationProvider dateAdapter={AdapterLuxon}>
+    <ThemeProvider theme={theme}>
+      <TestRouter router={router}>
+        <GqlMockedProvider>
+          <AppealsWrapper>
+            <AppealProgressBar
+              given={given}
+              received={received}
+              committed={committed}
+              amount={amount}
+              amountCurrency={amountCurrency}
+            />
+          </AppealsWrapper>
+        </GqlMockedProvider>
+      </TestRouter>
+    </ThemeProvider>
+  </LocalizationProvider>
+);
+
+describe('AppealProgressBar', () => {
+  describe('Errors', () => {
+    it('handle string instead of number', async () => {
```

Comment:
Is handling strings necessary? If so, I'd recommend changing the types in `<AppealProgressBar>` to `number | string`.

Reply:
It shouldn't be; I just wrote that test to test in case an amount in the DB was a string value.

Reply:
Gotcha. The type safety of GraphQL will guarantee that the types coming from the API are what we expect. It will throw errors if they don't. So I'm not sure that this is a useful test. Do you disagree?